.eslintrc.js

```diff
@@ -22,8 +25,8 @@ module.exports = {
     "WXEnvironment": true
   },
   rules: {
-    'no-console': process.env.NODE_ENV !== 'production' ? 0 : 2,
-    'no-useless-escape': 0,
-    'no-empty': 0
+    'no-console': process.env.NODE_ENV !== 'production' ? OFF : ERROR,
+    'no-useless-escape': OFF,
```

Comment:
```suggestion
    'no-useless-escape': 'off',
```

Reply:
I made all suggested changes in this PR #10551. (Unfortunately, I deleted the previous fork of the project and at the moment I do not have a possibility to edit this PR)